manim/animation/composition.py

```diff
@@ -46,8 +46,8 @@ def __init__(
         super().__init__(self.group, rate_func=rate_func, lag_ratio=lag_ratio, **kwargs)
         self.run_time: float = self.init_run_time(run_time)
 
-    def get_all_mobjects(self) -> Group:
-        return self.group
+    def get_all_mobjects(self) -> Sequence[Mobject]:
+        return list(self.group)
```

Comment:
isn't this a breaking change? maybe add it to changelog?

Reply:
You are right, I'll add that. However, I don't think anyone used that that method returned a group anywhere as it is a quite low level method.